catalog/spatial/registry/registry-publication-update-handler/src/main/java/org/codice/ddf/registry/publication/RegistryPublicationHandler.java

```diff
@@ -103,7 +103,7 @@ private void processUpdate(Metacard mcard) {
                     return null;
                 });
             } catch (PrivilegedActionException e) {
-                LOGGER.debug("Unable to send update for {}", mcard.getTitle());
+                LOGGER.error("Unable to send update for {}", mcard.getTitle());
```

Comment:
Anything at warn or above needs to tell an administrator what they can do to resolve the issue.